server/routes/notification.js

```diff
@@ -0,0 +1,19 @@
+const { Route53Resolver } = require("aws-sdk");
+const express = require("express");
+const router = express.Router();
+const {
+  getAllNotifications,
+  getAllUnreadNotifications,
+  createNewNotification,
+  markNotificationAsRead,
+} = require("../controllers/notifications");
+// works
+router.route("/").get(getAllNotifications);
+//works
+router.route("/unread").get(getAllUnreadNotifications);
+//works
+router.route("/new").post(createNewNotification);
+//works
+router.route("/markread").patch(markNotificationAsRead);
```

Comment:
If you protect all these routes, you can easily do the functionality I mention earlier about fetching by user